modules/videoio/cmake/detect_ffmpeg.cmake

```diff
@@ -34,7 +34,7 @@ if(NOT HAVE_FFMPEG AND PKG_CONFIG_FOUND)
 
     # rewrite libraries to absolute paths
     foreach(lib ${FFMPEG_LIBRARIES})
-      find_library(FFMPEG_ABSOLUTE_${lib} "${lib}" PATHS "${FFMPEG_lib${lib}_LIBDIR}" NO_DEFAULT_PATH)
```

Comment:
Dump values of:
- `CMAKE_FIND_ROOT_PATH`
- `${FFMPEG_lib${lib}_LIBDIR}`
- your `FFMPEG_INSTALL_PREFIX`

Reply:
```
CMAKE_FIND_ROOT_PATH: /home/user/Android/android-ndk-r18b
-- FFMPEG_LIBRARY_DIRS: /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/lib
-- FFMPEG_libavformat_LIBDIR: /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/lib
-- FFMPEG_libavcodec_LIBDIR: /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/lib
-- FFMPEG_libz_LIBDIR: 
-- FFMPEG_libswscale_LIBDIR: /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/lib
-- FFMPEG_libavutil_LIBDIR: /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/lib
-- FFMPEG_libm_LIBDIR: 
-- FFMPEG_libavresample_LIBDIR: /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/lib
-- FFMPEG_libavutil_LIBDIR: /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/lib
-- FFMPEG_libm_LIBDIR: 
```

```
FFMPEG_INSTALL_PREFIX /home/user/android-install/ffmpeg-arm64-v8a-n4.0.3
tree of FFMPEG_INSTALL_PREFIX:
/home/user/android-install/ffmpeg-arm64-v8a-n4.0.3/
├── include
│   ├── libavcodec
│   │   ├── ac3_parser.h
│   │   ├── adts_parser.h
│   │   ├── avcodec.h
│   │   ├── avdct.h
│   │   ├── avfft.h
│   │   ├── d3d11va.h
│   │   ├── dirac.h
│   │   ├── dv_profile.h
│   │   ├── dxva2.h
│   │   ├── jni.h
│   │   ├── mediacodec.h
│   │   ├── qsv.h
│   │   ├── vaapi.h
│   │   ├── vdpau.h
│   │   ├── version.h
│   │   ├── videotoolbox.h
│   │   ├── vorbis_parser.h
│   │   └── xvmc.h
│   ├── libavdevice
│   │   ├── avdevice.h
│   │   └── version.h
│   ├── libavfilter
│   │   ├── avfilter.h
│   │   ├── buffersink.h
│   │   ├── buffersrc.h
│   │   └── version.h
│   ├── libavformat
│   │   ├── avformat.h
│   │   ├── avio.h
│   │   └── version.h
│   ├── libavresample
│   │   ├── avresample.h
│   │   └── version.h
│   ├── libavutil
│   │   ├── adler32.h
│   │   ├── aes_ctr.h
│   │   ├── aes.h
│   │   ├── attributes.h
│   │   ├── audio_fifo.h
│   │   ├── avassert.h
│   │   ├── avconfig.h
│   │   ├── avstring.h
│   │   ├── avutil.h
│   │   ├── base64.h
│   │   ├── blowfish.h
│   │   ├── bprint.h
│   │   ├── bswap.h
│   │   ├── buffer.h
│   │   ├── camellia.h
│   │   ├── cast5.h
│   │   ├── channel_layout.h
│   │   ├── common.h
│   │   ├── cpu.h
│   │   ├── crc.h
│   │   ├── des.h
│   │   ├── dict.h
│   │   ├── display.h
│   │   ├── downmix_info.h
│   │   ├── encryption_info.h
│   │   ├── error.h
│   │   ├── eval.h
│   │   ├── ffversion.h
│   │   ├── fifo.h
│   │   ├── file.h
│   │   ├── frame.h
│   │   ├── hash.h
│   │   ├── hmac.h
│   │   ├── hwcontext_cuda.h
│   │   ├── hwcontext_d3d11va.h
│   │   ├── hwcontext_drm.h
│   │   ├── hwcontext_dxva2.h
│   │   ├── hwcontext.h
│   │   ├── hwcontext_mediacodec.h
│   │   ├── hwcontext_qsv.h
│   │   ├── hwcontext_vaapi.h
│   │   ├── hwcontext_vdpau.h
│   │   ├── hwcontext_videotoolbox.h
│   │   ├── imgutils.h
│   │   ├── intfloat.h
│   │   ├── intreadwrite.h
│   │   ├── lfg.h
│   │   ├── log.h
│   │   ├── lzo.h
│   │   ├── macros.h
│   │   ├── mastering_display_metadata.h
│   │   ├── mathematics.h
│   │   ├── md5.h
│   │   ├── mem.h
│   │   ├── motion_vector.h
│   │   ├── murmur3.h
│   │   ├── opt.h
│   │   ├── parseutils.h
│   │   ├── pixdesc.h
│   │   ├── pixelutils.h
│   │   ├── pixfmt.h
│   │   ├── random_seed.h
│   │   ├── rational.h
│   │   ├── rc4.h
│   │   ├── replaygain.h
│   │   ├── ripemd.h
│   │   ├── samplefmt.h
│   │   ├── sha512.h
│   │   ├── sha.h
│   │   ├── spherical.h
│   │   ├── stereo3d.h
│   │   ├── tea.h
│   │   ├── threadmessage.h
│   │   ├── timecode.h
│   │   ├── time.h
│   │   ├── timestamp.h
│   │   ├── tree.h
│   │   ├── twofish.h
│   │   ├── version.h
│   │   └── xtea.h
│   ├── libswresample
│   │   ├── swresample.h
│   │   └── version.h
│   └── libswscale
│       ├── swscale.h
│       └── version.h
├── lib
│   ├── libavcodec.a
│   ├── libavdevice.a
│   ├── libavfilter.a
│   ├── libavformat.a
│   ├── libavresample.a
│   ├── libavutil.a
│   ├── libswresample.a
│   ├── libswscale.a
│   └── pkgconfig
│       ├── libavcodec.pc
│       ├── libavdevice.pc
│       ├── libavfilter.pc
│       ├── libavformat.pc
│       ├── libavresample.pc
│       ├── libavutil.pc
│       ├── libswresample.pc
│       └── libswscale.pc
└── share
    └── ffmpeg
        └── examples
            ├── avio_dir_cmd.c
            ├── avio_reading.c
            ├── decode_audio.c
            ├── decode_video.c
            ├── demuxing_decoding.c
            ├── encode_audio.c
            ├── encode_video.c
            ├── extract_mvs.c
            ├── filter_audio.c
            ├── filtering_audio.c
            ├── filtering_video.c
            ├── http_multiclient.c
            ├── hw_decode.c
            ├── Makefile
            ├── metadata.c
            ├── muxing.c
            ├── qsvdec.c
            ├── README
            ├── remuxing.c
            ├── resampling_audio.c
            ├── scaling_video.c
            ├── transcode_aac.c
            ├── transcoding.c
            ├── vaapi_encode.c
            └── vaapi_transcode.c

14 directories, 154 files
```